### PR TITLE
Setup CI/CD

### PR DESCRIPTION
**Pull request by jonesbusy**
Setup CI/CD

- [x]  This require a PR againts jenkins-infra/repository-permissions-updater: https://github.com/jenkins-infra/repository-permissions-updater/pull/3951
- [x] This require to create a job on ci.jenkins.io similar to https://ci.jenkins.io/job/Tools/job/plugin-installation-manager-tool/

Also I suggest to disable `Actions` until we reach a usable version of the tool (or we will create a lot of release for nothing).

But at least the CI will be ready and CD ready to be turn on in some weeks/month

### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -13,7 +13,8 @@
 
     <groupId>io.jenkins.plugin-modernizer</groupId>
     <artifactId>plugin-modernizer-parent-pom</artifactId>
-    <version>${revision}${changelist}</version>
```
Comment (sridamul) [inline pom.xml:16]: This is the version show when the jar is executed with -V option

**Review by sridamul — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (jonesbusy) [reply pom.xml:16]: Maven inherit the version.

Confirmed filtering works with 

```
java -jar plugin-modernizer-cli/target/jenkins-plugin-modernizer-6.ve1a_9d7f37c6f.jar --version
6.ve1a_9d7f37c6f
```

**Comment by NotMyFault** [comment]
> This require to create a job on ci.jenkins.io similar to [ci.jenkins.io/job/Tools/job/plugin-installation-manager-tool](https://ci.jenkins.io/job/Tools/job/plugin-installation-manager-tool/)

I've added a job for you, given this causes a reiteration over all repositories, I prefer to do that on the weekend :P

**Review by NotMyFault — APPROVED** [review]

**Comment by jonesbusy** [comment]
> > This require to create a job on ci.jenkins.io similar to [ci.jenkins.io/job/Tools/job/plugin-installation-manager-tool](https://ci.jenkins.io/job/Tools/job/plugin-installation-manager-tool/)
> 
> I've added a job for you, given this causes a reiteration over all repositories, I prefer to do that on the weekend :P

Thanks a lot!

**Pull request merged**